jerry-main/main-unix.c

```diff
@@ -485,6 +491,38 @@ main (int argc,
                         "Ignoring 'exec-snapshot' option because this feature isn't enabled!\n");
       }
     }
+    else if (!strcmp ("--save-literals-list-format", argv[i])
+             || !strcmp ("--save-literals-c-format", argv[i]))
+    {
+      if (++i >= argc)
+      {
+        jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Error: no file specified for %s\n", argv[i - 1]);
+        print_usage (argv[0]);
+        return JERRY_STANDALONE_EXIT_CODE_FAIL;
+      }
+
+      if (jerry_is_feature_enabled (JERRY_FEATURE_SNAPSHOT_SAVE))
+      {
+        is_save_literals_mode = true;
+        is_save_literals_mode_in_c_format_or_list = strlen ("--save-literals-c-format") == strlen (argv[i - 1]);
```

Comment:
Unfortunately this is looks less nicely. Please revert to the original, this is not time critical. Sorry for the extra work.